SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -110,8 +115,8 @@ final class SwedbankPaySDKViewModel: NSObject {
     /// - parameter errorCallback: called on failure
     /// - returns: Dictionary containing the endpoints in successCallback, `SwedbankPaySDK.Problem` in errorCallback
     private func getEndPoints(_ backendUrl: URL, successCallback: Closure<Dictionary<String, String>?>? = nil, errorCallback: Closure<SwedbankPaySDK.Problem>? = nil) {
-        sessionManager.request(backendUrl, method: .get, parameters: nil, encoding: JSONEncoding.default, headers: configuration?.headers).responseJSON(completionHandler: { response in
-            if let responseValue = response.result.value {
+        sessionManager.request(backendUrl, method: .get, parameters: nil, encoding: JSONEncoding.default, headers: configuration?.afHeaders).responseJSON(completionHandler: { response in
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 186 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)